.github/workflows/checks.yml

```diff
@@ -1,5 +1,9 @@
 name: Components Checks
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
```

Comment:
We need the `|| github.ref` to handle the non-PR case.

Reply:
updated it